app/src/docs/content/dataUnions/frameworkRoles.mdx

```diff
@@ -2,14 +2,14 @@ import { Link } from 'react-router-dom'
 import docsLinks from '$shared/../docsLinks'
 
 # Roles and responsibilities
-Each Data Union has one Admin (owner), and `n` (many) data producing members.
+Each Data Union has one Admin (owner), and one to many data producing members.
```

Comment:
The matemathical notation here was weird but can revert if it was intentional?

Reply:
```suggestion
Each Data Union has one Admin (owner), and potentially thousands or even millions of data producing members.
```
Guilty. Trying to be too cool. How about the above suggestion.

Reply:
Aiming high, even better :)